src/harness/reference_models/interference/interference.py

```diff
@@ -0,0 +1,648 @@
+  """
+
+  grant_objects = []
+
+  # Loop over each CBSD grant
+  for cbsd_data_record in cbsd_data_records:
+    registration = cbsd_data_record.get('registrationRequest')
+    grants = cbsd_data_record.get('grants')
+
+    # Check CBSD location
+    lat_cbsd = registration.get('installationParam', {}).get('latitude')
+    lon_cbsd = registration.get('installationParam', {}).get('longitude')
+    height_cbsd = registration.get('installationParam', {}).get('height')
+    height_type_cbsd = registration.get('installationParam', {}).get('heightType')
+    if height_type_cbsd == 'AMSL':
+      altitude_cbsd = terrainDriver.GetTerrainElevation(lat_cbsd, lon_cbsd)
+      height_cbsd = height_cbsd - altitude_cbsd
+
+    # Sanity check on CBSD antenna height
+    if height_cbsd < 1 or height_cbsd > 1000:
+      raise ValueError('CBSD height is less than 1m or greater than 1000m.')
+
+    for grant in grants:
+      # Return CBSD information
+      cbsd_grant = CbsdGrantInformation(
+        # Get information from the registration
+        latitude=lat_cbsd,
+        longitude=lon_cbsd,
+        height_agl=height_cbsd,
+        indoor_deployment=registration.get('installationParam', {})
+                                      .get('indoorDeployment'),
+        antenna_azimuth=registration.get('installationParam', {})
+                                    .get('antennaAzimuth'),
+        antenna_gain=registration.get('installationParam', {})
+                                 .get('antennaGain'),
+        antenna_beamwidth=registration.get('installationParam', {})
+                                      .get('antennaBeamwidth'),
+        cbsd_category=registration.get('cbsdCategory'),
+        max_eirp=grant.get('operationParam', {}).get('maxEirp'),
+        low_frequency=grant.get('operationParam', {})
+                        .get('operationFrequencyRange', {})
+                        .get('lowFrequency'),
+        high_frequency=grant.get('operationParam', {})
+                         .get('operationFrequencyRange', {})
+                         .get('highFrequency'),
+        is_managed_grant=is_managing_sas)
+      grant_objects.append(cbsd_grant)
+  return grant_objects
+
+def computeInterferencePpaGwpzPoint(cbsd_grant, constraint, h_inc_ant, 
+                                  max_eirp, region='SUBURBAN'):
+  """Compute interference grant causes to GWPZ or PPA protection area
+  
+  Routine to compute interference neighborhood grant causes to protection 
+  point within GWPZ or PPA protection area
+
+  Args:
+    cbsd_grant: a namedtuple of type CbsdGrantInformation
+    constraint: protection constraint of type ProtectionConstraint
+    h_inc_ant: reference incumbent antenna height (in meters)
+    max_eirp: The maximum EIRP allocated to the grant during IAP procedure
+    region: Region type of the GWPZ or PPA area
+  Returns:
+    interference: interference contribution(dBm)
+  """
+
+  if (cbsd_grant.latitude == constraint.latitude) and \
```

Comment:
Cleaner to use the parenthesis for line continuation:
```
 if (cbsd_grant.latitude == constraint.latitude and
    cbsd_grant.longitude == constraint.longitude):
```

Reply:
Updated   as per suggestion